index.js

```diff
@@ -6,6 +6,7 @@ var jsonfile = require("jsonfile");
 var setup = jsonfile.readFileSync("./setup.json");
 var serverConfig = jsonfile.readFileSync("./config/airvantage/" + setup.dataCenter + ".json");
 var credentials = _.extend({}, setup.credentials, serverConfig.credentials);
+var companyUid = _.extend({}, setup.credentials, serverConfig.companyUid);
```

Comment:
Not needed, the `companyUid` is already available through the `setup` variable.
I'll change that.